src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -143,12 +144,11 @@
                     >
                       CHANGED
                     </v-chip>
+
                     <!-- all other states -->
-                    <template
-                      v-for="(action, i) in orgPerson.actions"
-                      v-else
-                    >
```

Comment:
eslint didn't like the v-for in the template element so I moved it to the v-chip below. I kept the template element because it looked weird to have both a v-else and a v-for in the v-chip.